doc/source/whatsnew/v0.18.0.txt

```diff
@@ -205,7 +205,8 @@ other anchored offsets like ``MonthBegin`` and ``YearBegin``.
 Other API Changes
 ^^^^^^^^^^^^^^^^^
 
-
+- DataFrame between_time now only parses a fixed set of time strings.  Parsing
```

Comment:
use double backticks ``DataFrame.between_time`` (though this also applies to ``Series``, yes? I would put a mini-example here as well.